crates/bevy_animation/src/lib.rs

```diff
@@ -565,8 +565,17 @@ fn run_animation_player(
     }
 }
 
-/// Update `weights` based on weights in `keyframes` at index `key_index`
-/// with a linear interpolation on `key_lerp`.
+/// Update `weights` based on weights in `keyframe` with a linear interpolation
+/// on `key_lerp`.
+fn lerp_morph_weights(weights: &mut [f32], key_lerp: f32, keyframe: impl Iterator<Item = f32>) {
```

Comment:
Nit: could we swap the two arguments.
```suggestion
fn lerp_morph_weights(weights: &mut [f32], keyframe: impl Iterator<Item = f32>, key_lerp: f32) {
```